internal_transfers/actions/src/parse.ts

```diff
@@ -91,3 +97,27 @@ export function tryParseERC20Transfer(log: Log): TransferEvent | null {
     amount: BigInt(value),
   };
 }
+
+export function tryParseWethAction(log: Log): TransferEvent | null {
+  if (log.address !== WETH_TOKEN_ADDRESS) return null; // Not emitted by WETH Contract!
+  const eventLog = wethInterface.parseLog(log);
```

Comment:
@fedgiac -- here is the change you suggested, but I do not understand your new question.